amf-client/jvm/src/test/scala/amf/client/validation/JvmPayloadValidationTest.scala

```diff
@@ -41,4 +37,14 @@ class JvmPayloadValidationTest extends ClientPayloadValidationTest with NativeOp
       report.results.asSeq.head.message shouldBe "Regex defined in schema could not be processed"
     }
   }
+
+  test("Validation against a number with multipleOf 0 should throw error") {
+    amf.Core.init().asFuture.flatMap { _ =>
+      amf.Core.registerPlugin(PayloadValidatorPlugin)
+      val shape     = new ScalarShape().withDataType(DataTypes.Number).withMultipleOf(0)
+      val validator = shape.payloadValidator("application/json").asOption.get
+      val report    = validator.syncValidate("application/json", "0")
```

Comment:
I would add another example with a different number. 0/0 is a special (?) division. Or modify it.

Reply:
nice catch, added another line here to validate against a positive number